internal/website/content/howto/docstore/actions.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Actions"
```

Comment:
Optional: Existing how-to guide titles tend to be a verb phrase like "Publish Messages to a Topic". Consider changing the title to "Reading and Writing Documents" or something similar.

Reply:
Done.